test/aarch64/configs/ubuntu.yaml

```diff
@@ -1,5 +1,5 @@
 kernelversion: 199
-kernelrelease: 4.15.0-188
+kernelrelease: 4.15.0-188-generic
```

Comment:
Hmm, not sure why we're having to change existing test input data like this? To be backwards compatible, I'd think that we'd want to be able to support this new format (with `-generic` in `kernelrelease`), and the old format (without `-generic`).

Reply:
I can change back. Ideally, we won't see any without `-{flavor}`, but it is good to check for backwards compatibility.